test/unit/util/test_sqlite_utils.py

```diff
@@ -37,19 +39,14 @@ def test_sqlite_exploits():
 
 def __assert_has_n_rows(connection, query, n):
     count = 0
-    for row in connection.cursor().execute(query):
+    for _ in connection.cursor().execute(query):
         count += 1
     assert count == n
 
 
 def __assert_query_errors(connection, query):
-    exception = False
-    try:
-        for row in connection.cursor().execute(query):
-            pass
-    except Exception:
-        exception = True
-    assert exception
+    with pytest.raises(Exception):
```

Comment:
This is only a small point but I don't think we shouldn't introduce pytest as a dependency to unit tests just to cleanup this pattern IMO. The cost of introducing the hard dependency on the library outweighs cleaning up the test a bit. I'm going merge this, but I'm going to keep resisting pytest.raises unless I have another reason to import pytest.